### DISCUSSION TITLE
Multiselect group can't set selected values,

### DISCUSSION THREAD

**Discussion by sathishcst10** [primefaces/community#1244] [PrimeNG] [answered]
Hi
While initial load there is a api call happening, and its return some default selected value for multiselect group, I just forward the vaues to multiSelect group, but its  selected and not showing label on the chip.

Please look at the code in stackblitz
['StackBlitz'](https://stackblitz.com/edit/ap4ngt?file=src%2Fapp%2Fdemo%2Fmulti-select-group-demo.html)

example image
![Untitled](https://github.com/primefaces/community/assets/6767773/97c182c1-f22f-4423-89af-8bfafab6e154)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Here is a refactored and working version of your example, please see the comment lines https://stackblitz.com/edit/ap4ngt-m9dntd?file=src%2Fapp%2Fdemo%2Fmulti-select-group-demo.ts,src%2Fapp%2Fdemo%2Fmulti-select-group-demo.html
